js/classes/entities/foreground/hud.js

```diff
@@ -149,10 +149,9 @@ class Hud {
             "5"
         ));
 
-        // this.addComponent("DashCharge", new DashCharge(
-        //     new Vector(Camera.SIZE.x - 100, Camera.SIZE.y - 100),
-
-        // ));
+        this.addComponent("dashCooldown", new DashCooldown(
+            new Vector(Camera.SIZE.x - 700, Camera.SIZE.y - 100)
```

Comment:
Would it be better to use a positive offset from the right edge of the hotbar for the x position (something like `hotbarXStart + 5 * (hotbarItemWidth + 4) + 100`), instead of a negative offset from the edge of the screen?